docs/howto/hooks/lua.md

```diff
@@ -233,6 +233,11 @@ Sets the object at the given bucket and key to the value of the supplied value s
 Deletes the object at the given key  
 `path_uri` - A valid Azure blob storage uri in the form of `https://myaccount.blob.core.windows.net/mycontainer/myblob`
 
+### `azure/abfss_transform_path(path)`
```

Comment:
1. Please update [delta_exporter.export_delta_log](https://docs.lakefs.io/howto/hooks/lua.html#lakefscatalogexportdelta_exporterexport_delta_logaction-table_def_names-write_object-delta_client-table_descriptors_path) function doc reference. 

1. assuming most users will use the delta_exporter first and stumble with this issue, please add a bullet in [unity_exporter.register_tables](https://docs.lakefs.io/howto/hooks/lua.html#lakefscatalogexportunity_exporterregister_tablesaction-table_descriptors_path-delta_table_details-databricks_client-warehouse_id) mentioning this.